charts/collector/requirements.yaml

```diff
@@ -0,0 +1,7 @@
+dependencies:
+- name: postgresql
+  version: 8.2.1
+  repository: https://kubernetes-charts.storage.googleapis.com/
```

Comment:
You should use a condition: https://github.com/Kong/charts/blob/master/charts/kong/requirements.yaml#L5